components/admin/events/events-page.tsx

```
@@ -1259,9 +1259,9 @@ function EventDetailView({
 							<ul className="space-y-1">
 								{detail.views.map((view) => (
 									<li className="text-sm" key={view.id}>
-										{view.userName} ({view.userEmail}){" "}
+										{view.userName}{" "}
 										<span className="text-muted-foreground text-xs">
-											{view.viewedAt.toLocaleDateString()}
+											{"Last seen at: "}{view.viewedAt.toLocaleDateString()}
```

Biome (2.4.10)

[error] Should not have curly braces around expression.

(lint/style/useConsistentCurlyBraces)